blockchain/binance/api/client.go

```diff
@@ -33,6 +35,11 @@ func (c *Client) GetTransactionsByAddress(address string, limit int) ([]Tx, erro
 
 	var result TransactionsResponse
 
-	err := c.req.Get(&result, "bc/api/v1/txs", params)
+	_, err := c.req.Execute(context.Background(), client.NewReqBuilder().
```

Comment:
`context.TODO()` should be more accurate here.

Reply:
@iCell fixed